.buildbot.sh

```diff
@@ -2,7 +2,41 @@
 
 set -eu
 
-pipx install rebench
+# Install Rust
+CARGO_HOME="$(pwd)/.cargo"
+export CARGO_HOME
+RUSTUP_HOME="$(pwd)/.rustup"
+export RUSTUP_HOME
+export RUSTUP_INIT_SKIP_PATH_CHECK="yes"
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+sh rustup.sh --default-host x86_64-unknown-linux-gnu \
+    --default-toolchain nightly \
+    --no-modify-path \
+    --profile default \
+    -y
+export PATH="${CARGO_HOME}"/bin/:"$PATH"
+
+# Set up yk.
+#
+# We don't yet use a cached ykllvm because at the time of writing we only cache
+# a release+assertions build and we don't want to benchmark assertions.
```

Comment:
Why is this a problem? We're not benchmarking the time it takes to build yklua are we? By the time the binary is built, and assertions in ykllvm are irrelevant I think, unless assertions are embedded *in* yklua?

Reply:
Ah yes. It's burnt into my head that we are using the LLVM API at runtime, but we are not!

OK, so should my next PR be to use the cached ykllvm? Part of me still wants to build it from scratch, but I don't feel strongly.

Reply:
It's fine not to use a cached ykllvm here if you feel strongly about it, but let's nuke the comment either way.

Reply:
I've killed the comment in c0307280146d0ff0d881892ca3b8b61295f549e4